robottelo/cli/factory.py

```diff
@@ -1788,6 +1789,79 @@ def make_template(options=None):
     return create_object(Template, args, options)
 
 
+@cacheable
+def make_variable(options=None):
```

Comment:
The only helper which is still not renamed yet :) All the rest are smart variables, and even corresponding hammer command is `hammer smart-variable create`, so please make the naming consistent.